modules/executors/main.tf

```
@@ -31,6 +31,8 @@ locals {
     }
   }
 
+  # if using local SSDs and using the default value of 100, lower it to 10, otherwise use the configured value either way.
+  boot_disk_size = var.use_local_ssd ? var.boot_disk_size == 100 ? 10 : var.boot_disk_size : var.boot_disk_size
```

```suggestion
  boot_disk_size = var.use_local_ssd ? (var.boot_disk_size == 100 ? 10 : var.boot_disk_size) : var.boot_disk_size
```

Nit, but makes it a bit more readable